tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java

```diff
@@ -3081,7 +3094,7 @@ public String getWinningPath(String path) {
     public Collection<String> getExtraPaths() {
         Set<String> toAddTo = new HashSet<>();
         toAddTo.addAll(getRawExtraPaths());
-        for (String path : this) {
+        for (String path : this.iterableWithoutExtras()) {
```

Comment:
java.lang.StackOverflowError would occur here due to infinite loop if extras were included